core/templates/dev/head/pages/exploration_editor/simple_editor_tab/simple_editor_sidebar_directive.html

```diff
@@ -134,4 +140,10 @@
     text-indent: 30px;
     width: 220px;
   }
+  .material-icons.md-18 {
```

Comment:
Here you override [this](https://github.com/oppia/oppia/blob/simple-editor/core/templates/dev/head/css/oppia.css#L71), which causes all icons with this class to change the color, use something like 'simple-editor-sidebar .error-outline-icon'